lib/heuristics/concerns/scheduling_data_initialisation.rb

```diff
@@ -311,4 +309,17 @@ def compute_capacities(quantities, vehicle, available_units = [])
 
     capacities
   end
+
+  def exist_common_multiplexer(lapses)
```

Comment:
Could you add some comments on the purpose of the method ?

if a boolean is returned, it would be more natural to add a question mark to the method name : exist_common_multiplexer?

Does it return true if  multiplexers.size is equal to zero ?